test/WebSites/BasicWebSite/project.json

```diff
@@ -15,5 +15,10 @@
         "dnx451": { },
         "dnxcore50": { }
     },
+    "exclude": [
+        "wwwroot",
+        "node_modules",
+        "bower_components"
+    ],
     "webroot": "wwwroot"
```

Comment:
Remove this line. No modern `project.json` file should contain a `webroot` setting.